From b8df63b02de2ffcbef4a88bb9ea404c1d8dd8802 Mon Sep 17 00:00:00 2001
From: kasiawasiuta <kasiawasiuta@users.noreply.github.com>
Date: Tue, 10 Mar 2020 11:40:42 +0100
Subject: [PATCH 04/23] Merge pull request #1 from
 kasiawasiuta/README-modification

Modified doc files to match the project name and objectives
---
 CONTRIBUTING | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CONTRIBUTING b/CONTRIBUTING
index f676ec774..06c889778 100644
--- a/CONTRIBUTING
+++ b/CONTRIBUTING
@@ -15,7 +15,7 @@ to accomplish and why. Use cases are important for features to be accepted.
 Here you'll see if there is consensus about your idea.
 
 2. If in step 1 you get an acknowledgment from the project leaders, use the
-   following procedure to submit a patch:D
+   following procedure to submit a patch:
 
     a. Fork MemKeyDB on github ( http://help.github.com/fork-a-repo/ )
     b. Create a topic branch (git checkout -b general-change-topic)
-- 
2.19.1.6.gb485710b

